conf/app.routes

```diff
@@ -150,9 +150,15 @@ GET          /how-to-claim-refund-external                                contro
 GET          /agents/how-to-claim-refund-external                         controllers.NotMigratedUserController.redirectAgent()
 
 # Income Sources
-GET          /income-sources/manage/choose-income-sources                 controllers.ManageIncomeSourceController.show()
-GET          /agents/income-sources/manage/choose-income-sources          controllers.ManageIncomeSourceController.showAgent()
-GET          /income-sources/add/new-income-sources                       controllers.AddIncomeSourceController.show()
-GET          /agents/income-sources/add/new-income-sources                controllers.AddIncomeSourceController.showAgent()
-GET          /income-sources/cease/cease-an-income-source                 controllers.CeaseIncomeSourceController.show()
-GET          /agents/income-sources/cease/cease-an-income-source          controllers.CeaseIncomeSourceController.showAgent()
\ No newline at end of file
+GET          /income-sources/manage/choose-income-sources                                               controllers.ManageIncomeSourceController.show()
```

Comment:
again, optional, you can separate these which are not yet implemented in a separate section, but it really up to you